Maddori.Apple/Maddori.Apple/Screen/MyBox/Cell/MyFeedbackCollectionViewCell.swift

```diff
@@ -67,4 +79,8 @@ final class MyFeedbackCollectionViewCell: BaseCollectionViewCell {
         titleLabel.text = title
         contentLabel.setTextWithLineHeight(text: content, lineHeight: 22)
     }
+    
+    func setHiddenDivider(value: Bool) {
+        dividerView.isHidden = value
+    }
```

Comment:
p5;
그냥 슥 보다가 든 생각인데 setHiddenDivider로 네이밍을 하게 되면 
hidden 인 divider를 보이게 한다는건지, divider를 hidden하게 한다는건지 명확하게 안와닿아서 리뷰 남겨봅니다 ^___^

value가 true 면 hidden이 되도록 하려는 거라면,
setDividerHidden()으로 두면 어떨까요??
그럼 함수 실행할 때에도 setDividerHidden(true) 이렇게 되니까
아! divider가 hidden이 되겠구나! 가 돼서 이해가 잘 될 것 같았어요 ㅎㅎ..

Reply:
오 대신
```suggestion
    func setDividerHidden(_ value: Bool) {
        dividerView.isHidden = value
    }
```
저 함수 안에 입력값을 _로 두면 더 가독성이 좋아질 것 같다는 생각입니다 ㅎㅎ
함수 사용할때 `setDivdierHidden(true)`가 되니까요!

Reply:
와우 ! 파라미터를 생략하는걸 어느 때 써야 적절한건지 항상 명확한 답이 안나와서 주저리주저리 쓰고 있었는데, 이드의 예시를 보니 단번에 납득이 됐군요!! 
너무 감사합니다 ^_______^